src/routes/index.js

```diff
@@ -7,12 +7,14 @@ import About from "../pages/about";
 import Research from "../pages/research";
 import FAQ from "../pages/faq";
 import Methods from "../pages/methods";
+import Comparison from "../pages/comparison";
 
 const Routes = class Routes extends Component {
   render() {
     return (
       <Switch>
         <Route exact path="/" component={Main} />
+        <Route exact path="/comparison" component={Comparison} />
```

Comment:
This conflicts with the work in #84. routes/index.js and Main will [swap positions](https://github.com/CxSci/SA-dashboard/pull/84/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67L18-R21) in the component hierarchy, so routes/index.js will end up being just for components which need to render in addition to Main rather than instead of it.

Additionally, given my comment about ComparisonSidebar.js, you'll want /comparison routing to happen inside of sidebar.js anyhow.